### PR TITLE
AST-1466 - Nav menu is not working properly on the sticky header

### PR DESCRIPTION
**Pull request by shubshinde**
### Description
The header nav menu was not scrollable when expanded menu items in tablet & mobile view,
Added CSS to Scroll Menu items when the main nav menu was set to sticky.

### Screenshots
Before Fix (Not Scrollable)  - https://d.pr/v/jKs25S
After Fix (Is Scrollable)  - https://d.pr/v/OFzaDw

### Types of changes
Bugfix 

### How has this been tested?
This has been tested in mobile and Tablet view, For reproducing this I have added many sub-menu items to the menu, then tested after expanding them to check if it's scrolling.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/css/unminified/frontend.css [hunk 1]
```diff
@@ -2746,6 +2746,16 @@ body .entry-content .wp-block-latest-posts li {
...
+.ast-primary-sticky-header-active.ast-main-header-nav-open .ast-builder-menu .main-navigation ul {
+  overflow: scroll;
+  height: 55vh;
+}
+
+.ast-primary-sticky-header-active.ast-main-header-nav-open {
+  overflow: scroll;
+  height: 10vh;
+}
+
```
Comment (nileshc-bsf) [inline assets/css/unminified/frontend.css:2758]: @shubshinde - Is this specific for Mobile/tablet?
Cross-check for desktop, it should not affect the menu height.

Comment (shubshinde) [reply assets/css/unminified/frontend.css:2759]: Added Media Query for tablet, mobile devices.

File: sass/site/header/_header-builder-main.scss [hunk 2]
```diff
@@ -183,3 +183,19 @@
...
+		&.ast-main-header-nav-open{
+			.ast-builder-menu{
+				.main-navigation{
+					overflow: scroll;
+					height: 70vh;
+				}
+			}
+		}
+	}
+}
```
Comment (premanshup) [inline sass/site/header/_header-builder-main.scss:199]: @shubshinde Can you please move this CSS to Astra Pro - Sticky Header - CSS.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: sass/site/header/_header-builder-main.scss (hunk 2 above)
Comment (shubshinde) [reply sass/site/header/_header-builder-main.scss:199]: Ok

**Comment by shubshinde** [comment]
Closing this PR, as this code is moved to Addons.

https://github.com/brainstormforce/astra-addon/pull/567

**Pull request closed without merging**